cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,36 @@
+/* eslint-disable max-len */
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  it('Should be fill firstName', () => {
+    const name = 'Roma';
+    const lastName = 'Mykytiuk';
+    const email = 'mykytiuk@gmail.com';
+    const curentAdres = Math.floor(Math.random() * 100);
```

Comment:
The 'current address' is being filled with a random number, which might not be ideal for testing purposes. Consider using a consistent address format to ensure reliable test results.